lib/engine/game/g_18_norway/map.rb

```diff
@@ -0,0 +1,327 @@
+            ['E25'] => 'border=edge:0,type:divider;border=edge:1,type:divider;border=edge:2,type:divider;'\
+                       'border=edge:3,type:divider;border=edge:5,type:divider',
+            ['F22'] => 'border=edge:1,type:divider;border=edge:2,type:divider;border=edge:3,type:divider;'\
+                       'border=edge:4,type:divider;border=edge:5,type:divider',
+            ['F24'] => 'border=edge:0,type:divider;border=edge:2,type:divider;border=edge:4,type:divider;'\
+                       'border=edge:5,type:divider',
+            ['I21'] => 'town=revenue:20;path=a:0,b:_0;path=a:2,b:_0',
+          },
+          red: {
+            ['C37'] => 'offboard=revenue:yellow_30|green_50|brown_60;path=a:3,b:_0,track:narrow',
+            ['H34'] => 'offboard=revenue:yellow_20|green_40|brown_60|gray_80;path=a:3,b:_0',
+            ['I19'] => 'offboard=revenue:yellow_20|green_40|brown_60;path=a:1,b:_0',
+            ['I29'] => 'offboard=revenue:yellow_20|green_40|brown_60;path=a:1,b:_0;path=a:2,b:_0',
+            ['K1'] => 'offboard=revenue:green_0|brown_100;path=a:0,b:_0',
+          },
+          yellow: {
+            ['B26'] => 'label=B;city=revenue:20;path=a:1,b:_0,track:narrow;path=a:2,b:_0,track:narrow;path=a:4,b:_0;'\
+                       'border=edge:0,type:impassable;border=edge:5,type:impassable',
+            ['B32'] => 'label=S;city=revenue:20;path=a:1,b:_0,track:narrow;path=a:2,b:_0,track:narrow;path=a:4,b:_0;'\
+                       'border=edge:3,type:impassable',
+            ['C19'] => 'label=Å;town=revenue:10;path=a:5,b:_0;path=a:1,b:_0,track:narrow;path=a:3,b:_0,track:narrow;'\
+                       'border=edge:4,type:impassable',
+            ['C35'] => 'label=K;city=revenue:20;path=a:0,b:_0,track:narrow;path=a:1,b:_0,track:narrow;path=a:4,b:_0',
+            ['D18'] => 'label=M;city=revenue:20;path=a:5,b:_0;path=a:2,b:_0,track:narrow;path=a:3,b:_0,track:narrow;'\
+                       'border=edge:1,type:impassable',
+            ['G17'] => 'label=T;city=revenue:20;path=a:0,b:_0;path=a:3,b:_0,track:narrow',
+            ['G29'] => 'label=O;city=revenue:30;city=revenue:30;city=revenue:30;path=a:0,b:_0;path=a:2,b:_1;path=a:4,b:_2',
+          },
+          blue: {
+            %w[C1 E1 G1 I1] => '',
+            %w[A1 A3 A5 A7 A9 A11 A13 A15 A17 A19] => '',
+            %w[D36 E35 F34 F36 G33 G35 H36] => '',
+            %w[B2 D2 F2 H2 J2] => '',
+            %w[C3 E3 G3 I3] => '',
+            %w[B4 D4 F4 H4 J4] => '',
+            %w[C5 E5 G5 I5] => '',
+            %w[B6 D6 F6 H6 J6] => '',
+            %w[C7 E7 G7 I7] => '',
+            %w[B8 D8 F8 H8] => '',
+            %w[C9 E9 G9 I9] => '',
+            %w[B10 D10 F10 H10] => '',
+            %w[C11 E11 G11] => '',
+            %w[B12 D12 F12 H12] => '',
+            %w[C13 E13 G13] => '',
+            %w[B14 D14] => '',
+            %w[C15] => '',
+            %w[B16 B18] => '',
```

Comment:
This map might look better with less blue hexes on it, just keeping the ones immediately next to the land.

Reply:
![image](https://github.com/tobymao/18xx/assets/8018982/0df75429-bbd3-4320-bc7e-8d4884e586dd)
I think it looks strange if I remove them, do you still prefer to remove them?

Reply:
Yeah, that's not great either. I didn't like the huge block of blue with the original version, it drowns out the rest of the map. Would it be any better if you just had a single row of water hexes along the coast, in hexes J2, J4, J6, I7, …?
But this is purely aesthetic, go with whatever you want.

Reply:
Let us try with single row

Reply:
Updated

Reply:
I prefer this version with the single set of water hexes along the coast. But go back to the version with more blue hexes if that is the one that you like.

Reply:
Let's go with single set of water hexes